### ISSUE TITLE
Reverse the Polar Plot

### ISSUE THREAD

**Issue by josephholler** [ghtmtt/DataPlotly#88]
Enhancement suggestion: Is it possible to reverse the direction of the Polar Plot? It goes counter-clockwise, but for charting a value and direction, it would be ideal to have 0 degrees at top and increment clock-wise for 90 degrees East, 180 degrees south, etc.

**Comment by ghtmtt** [comment]
Yep this could be an useful enhancement and luckily plotly provides this possibility. I'll put here a code snipped as a reminder, will work on that soon:

```
import plotly
import plotly.graph_objs as go

vl = iface.activeLayer()

f1 = []
for i in vl.getFeatures():
    f1.append(i['PH'])
    
f2 = []
for i in vl.getFeatures():
    f2.append(i['EC'])


trace1 = go.Scatterpolar(
r = f1,
theta = f2,
subplot='polar',
mode='markers'
)

layout = go.Layout(
    polar = dict(
    angularaxis = dict(
    rotation = 90,
    direction = "clockwise"
    )
    )
)

data = [trace1]
fig = go.Figure(data=data, layout = layout)
plotly.offline.plot(fig)

```

![image](https://user-images.githubusercontent.com/2884884/53298206-d849ae00-382a-11e9-8d68-ce69aa6ff3c5.png)

**Comment by ghtmtt** [comment]
fixed in #121 thanks @josephholler
